ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -522,6 +525,25 @@ private int executeIncrementalLoad(DriverContext driverContext) {
       // bootstrap of tables if exist.
       if (builder.hasMoreWork() || work.getPathsToCopyIterator().hasNext() || work.hasBootstrapLoadTasks()) {
         DAGTraversal.traverse(childTasks, new AddDependencyToLeaves(TaskFactory.get(work, conf)));
+      } else if (work.dbNameToLoadIn != null) {
```

Comment:
I think, work.dbNameToLoadIn will be null if you don't specify the name in REPL LOAD command. In this case, we should get the name from DumpMetadata to set the last repl ID.

Reply:
Ok. Thanks for bringing that up. Done.